_includes/macros.html

```diff
@@ -285,3 +290,62 @@ <h1 class="h5 {{'u-visually-hidden' if hide_heading}}">Share this page</h1>
 
 {% endmacro %}
 
+{# ==========================================================================
+
+   youtube()
+
+   ==========================================================================
+
+   Description:
+
+   Builds Youtube markup when given:
+
+   options (optional): An object used to customize the Youtube markup.
+
+   options.image_res: The image resolution to use for the Youtube image.
+
+   options.video_id: An id associated with a Youtube video.
+
+   ========================================================================== #}
+
+{% macro youtube(options) -%}
+    {% set image_res = options.image_res | default('default') %}
+    {% set video_id  = options.video_id  | default('en0Iq8II4fA') %}
+    {% set image_url = options.image_url | default('http://img.youtube.com/vi/' + video_id) %}
+    {% set video_url = 'http://www.youtube.com/embed/' + video_id %}
+    {% set img_res_options = {
+          'default' : 'default.jpg',
+          'standard': 'sddefault.jpg',
+          'medium'  : 'mqdefault.jpg',
+          'max'     : 'maxresdefault.jpg',
+          'high'    : 'hqdefault.jpg'
+        }
+    %}
+
+    <div class="youtube-player_container">
```

Comment:
Im not sure if we use other video sources. I think it's out of scope for this task but I think we could expand it to do that in the future.